buildpipeline/DotNet-CoreFx-Trusted-Linux-Crossbuild.json

```diff
@@ -273,6 +273,26 @@
         "failOnStandardError": "false"
       }
     },
+    {
+      "environment": {},
+      "enabled": true,
+      "continueOnError": true,
+      "alwaysRun": true,
+      "displayName": "Local copy MSBuild debug logs out of container",
+      "timeoutInMinutes": 0,
+      "condition": "succeededOrFailed()",
+      "task": {
+        "id": "d9bafed4-0b18-4f58-968d-86655b4d2ce9",
+        "versionSpec": "1.*",
+        "definitionType": "task"
+      },
+      "inputs": {
+        "filename": "docker",
+        "arguments": "run --rm -v $(Build.StagingDirectory):/docker_logs $(PB_DockerCommonRunArgs) find . -type f -name MSBuild_*.failure.txt -exec cp {} --target-directory=/docker_logs ;",
```

Comment:
Is it possible to combine this with the previous command rather than duplicate it? I think:

`find . -type f -name *.log -o -type f -name MSBuild_*.failure.txt ...`?

Reply:
Good tip, thanks